docs/DeveloperGuide.md

```diff
@@ -138,6 +138,66 @@ Use case ends.
   Use case resumes from step 1.
 
 ### Use case: UC05 - Add an internship application
```

Comment:
I think you have have missed out the UC05

Reply:
oh yeah, thanks for noticing :sweat_smile: